File: Python/3141.py
# Problema: 3141 - Dúvida Etária | Resposta: Accepted
# Linguagem: Python 3.11 [+1s]   | Tempo: 0.356s

from datetime import date

def pegaData():
    entrada = list(map(int, input().split('/')))
    return entrada[::-1]

nome = input()
dt_atual = date(*pegaData())
dt_nascimento = date(*pegaData())

anos = dt_atual.year - dt_nascimento.year
if(dt_atual.month < dt_nascimento.month 
   or (
       dt_atual.month == dt_nascimento.month 
       and dt_atual.day < dt_nascimento.day
    )):
    anos -= 1

if(dt_nascimento.strftime('%m-%d') == dt_atual.strftime('%m-%d')):
    print("Feliz aniversario!")

print(f"Voce tem {anos} anos {nome}.")

File: PHP/3141.php
<?php
// Problema: 3141 - Dúvida Etária | Resposta: Accepted
// Linguagem: PHP (8.1.2) [+1s]   | Tempo: 0.145s

[$nome, $dt_atual, $dt_nascimento] = [trim(fgets(STDIN)), trim(fgets(STDIN)), trim(fgets(STDIN))];
$dt_atual = DateTime::createFromFormat('d/m/Y', $dt_atual);
$dt_nascimento = DateTime::createFromFormat('d/m/Y', $dt_nascimento);

$anos = $dt_atual->diff($dt_nascimento)->y;

$saida = "";
if($dt_nascimento->format('m-d') == $dt_atual->format('m-d'))
    $saida = "Feliz aniversario!" . PHP_EOL;

$saida .= "Voce tem {$anos} anos {$nome}." . PHP_EOL;
print($saida);
?>